java/gen/src/findbugs/exclude.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<FindBugsFilter>
+
+</FindBugsFilter>
```

Comment:
Just a question. Do we need this empty file?

Reply:
yeah this was needed otherwise the analyze was throwing error

Reply:
Could you share the exceptions you hit?

Reply:
Try running 
```bash
mvn clean package -Panalyze
```

Then you get the error
```text
[ERROR] Could not find resource '--SNIP--orc/java/gen/src/findbugs/exclude.xml'. -> [Help 1]
```